src/test/java/sorinfoca/Tests/Tema8/TestPage/TestPages.java

```diff
@@ -0,0 +1,63 @@
+package sorinfoca.Tests.Tema8.TestPage;
+
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import sorinfoca.Tests.Tema8.pages.*;
+
+import java.time.Duration;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotEquals;
+
+public class TestPages extends BaseTest {
+
+    @Test(groups = {"desktop", "mobile"})
+    public void testBasicAjaxPage() {
+        BasicAjaxPage basicAjaxPage = new BasicAjaxPage(driver);
+        basicAjaxPage.goToPage();
+        basicAjaxPage.selectOption("Server");
+        basicAjaxPage.selectLanguage("Java");
+        basicAjaxPage.clickCodeItInButton();
+        assertEquals(basicAjaxPage.getSelectedOption(),"23");
+    }
+
+    @Test(groups = {"desktop", "mobile"})
+    public void javascriptRedirectPage() {
+        JavaScriptRedirectPage javascriptRedirectPage = new JavaScriptRedirectPage(driver);
+        javascriptRedirectPage.goToPage();
+        javascriptRedirectPage.clickRedirectButton(0);
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        wait.until(ExpectedConditions.urlContains("newpage"));
+        assertEquals("New Page", driver.getTitle());
```

Comment:
First value is the actual one and the second is the expected one. Please reverse order of the argument and change "New Page" with the actual title of the page. Thanks!